pkg/storage/business_case.go

```diff
@@ -109,7 +109,7 @@ func (s *Store) FetchBusinessCasesByEuaID(ctx context.Context, euaID string) (mo
 	return businessCases, nil
 }
 
-func createEstimatedLifecycleCosts(ctx context.Context, tx *sqlx.Tx, businessCase *models.BusinessCase) error {
+func createEstimatedLifecycleCosts(ctx context.Context, tx *hnysqlx.Tx, businessCase *models.BusinessCase) error {
```

Comment:
This is the single example of where the switch-out of `hnysqlx` for `sqlx` was at all apparent anywhere else in the code.